src/SensioLabs/Connect/Api/Api.php

```diff
@@ -67,50 +68,50 @@ public function getAccessToken()
         return $this->accessToken;
     }
 
-    public function get($url, $headers = array())
+    public function get($url, $headers = [])
     {
         $url = $this->constructUrlWithAccessToken($url);
 
         $this->logger->info(sprintf('GET %s', $url));
 
-        return $this->processResponse($this->browser->get($url, array_merge($headers, $this->getAcceptHeader())));
+        return $this->processResponse($this->browser->sendRequest(new Request('GET', $url, array_merge($headers, $this->getAcceptHeader()))));
     }
 
-    public function submit($url, $method = 'POST', array $fields, $headers = array())
+    public function submit($url, $method = 'POST', array $fields = [], $headers = [])
```

Comment:
please revert all unrelated VS changes - fixtures and XML files included

Reply:
They are reverted.